File: app/src/main/java/com/example/schoolairdroprefactoredition/cache/UserTokenCache.java
package com.example.schoolairdroprefactoredition.cache;

import com.example.schoolairdroprefactoredition.domain.DomainToken;

public class UserTokenCache {
    public static final String KEY = "imatoken";

    private DomainToken token;

    public DomainToken getToken() {
        return token;
    }

    public void setToken(DomainToken token) {
        this.token = token;
    }
}

File: app/src/main/java/com/example/schoolairdroprefactoredition/cache/UserTokenCache.kt
package com.example.schoolairdroprefactoredition.cache

import com.example.schoolairdroprefactoredition.domain.DomainToken

data class UserTokenCache(
        val token: DomainToken
) {
    companion object {
        const val KEY = "hahahahaa"
    }

    override fun toString(): String {
        return "UserTokenCache(token=$token)"
    }


}
